MobileCenter/MobileCenter/Internals/Sender/MSHttpSender.h

```diff
@@ -23,7 +22,12 @@ NS_ASSUME_NONNULL_BEGIN
 /**
  *  Pending http calls.
  */
-@property(atomic, strong) NSMutableDictionary<NSString *, id <MSSenderCall>> *pendingCalls;
+@property(atomic, strong) NSMutableDictionary<NSString *, MSSenderCall *> *pendingCalls;
+
+// TODO (jaelim): Add doc here.
+- (void)sendAsync:(NSObject *)container
```

Comment:
I would suggest to use a generic type on the class and then use it there instead of an `NSObject`. Then you'll be able to restrict the type on the `MSIngestionSender` class.